shuttering/prod/apply-to-adopt-a-child-placed-in-your-care-service-gov-uk.yml

```diff
@@ -9,7 +9,7 @@ shutter_all: false
 # - alias_target_resource_id
 shutter_all_a: false
 
-shutter_all_cname: false
+shutter_all_cname: true
```

Comment:
This dns zone uses A records for shuttering so we need shutter_all_a set to true and shutter_all_cname set to false as not required

Reply:
done 👍